specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2021-01-01-preview/purviewDataPolicy.json

```diff
@@ -0,0 +1,835 @@
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "policyName",
+            "description": "Id of an existing policy that needs to be fetched.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DataPolicy"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Policy Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DataPolicy_Get": {
+            "$ref": "./examples/DataPolicy_Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AttributeMatcher": {
+      "description": "Attribute matcher for a rule",
+      "type": "object",
+      "properties": {
+        "attributeName": {
+          "description": "AttributeName",
+          "type": "string"
+        },
+        "attributeValueIncludes": {
+          "description": "Value for attribute",
+          "type": "string"
+        },
+        "attributeValueIncludedIn": {
+          "description": "List of values for attribute",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "attributeValueExcludes": {
+          "description": "Value excluded for attribute",
+          "type": "string"
+        },
+        "attributeValueExcludedIn": {
+          "description": "List of values excluded for attribute",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "DnfCondition": {
+      "description": "The dnf Condition for a rule",
+      "type": "array",
+      "items": {
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/AttributeMatcher"
+        }
+      }
+    },
+    "CnfCondition": {
+      "description": "The cnf Condition for a rule",
+      "type": "array",
+      "items": {
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/AttributeMatcher"
+        }
+      }
+    },
+    "DecisionRule": {
+      "description": "The decision rule for a policy",
+      "type": "object",
+      "properties": {
+        "kind": {
+          "description": "The kind of rule",
+          "enum": [
+            "decisionrule"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "Kind",
+            "modelAsString": true
+          }
+        },
+        "effect": {
```

Comment:
action?

Reply:
A lot of our users are familiar with this term. Our team released APIs with similar payload few monthds ago.